config/application.rb

```diff
@@ -51,13 +51,8 @@ class Application < Rails::Application
 
     config.guidance_url = "https://www.gov.uk/government/publications/additional-payments-for-teaching-eligibility-and-payment-details"
 
-    if ENV["LOGSTASH_HOST"].present?
-      tcp_logger = LogStashLogger.new(type: :tcp,
-                                      host: ENV.fetch("LOGSTASH_HOST"),
-                                      port: ENV.fetch("LOGSTASH_PORT"),
-                                      ssl_enable: true)
-
-      SemanticLogger.add_appender(logger: tcp_logger, level: :info, formatter: :json)
-    end
+    config.log_tags = {
+      environment: ENV.fetch("ENVIRONMENT_NAME"),
```

Comment:
It might be best to add a default option to `.fetch` here, so `ENV.fetch("ENVIRONMENT_NAME", "local")` so it doesn't raise an error if `ENVIRONMENT_NAME` isn't set